week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,42 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isBetween5and10(number) {
+  if (number > 5 && number <= 10) {
+    return true;
+  }
+}
+var name = "Daniel";
+
+function isLongName(name) {
+  if (name.lenght > 5);
+  {
+    return true;
+  }
+}
+var theName = "Daniel";
+function startsWithD(theName) {
+  if (theName.includes("D")) {
```

Comment:
The name of your function doesn't match what it actually does. Your function checks if the name **includes** the letter "D", not if the name **starts with it** - those are different things :)